main/src/ca/uwaterloo/flix/language/phase/ClosureConv.scala

```diff
@@ -163,7 +163,8 @@ object ClosureConv {
       val v = convert(value)
       Expression.ArrayStore(b, i, v, tpe, loc)
 
-    case Expression.GetChannel(e, tpe, loc) =>
+    case Expression.GetChannel(exp, tpe, loc) =>
+      val e = convert(exp)
       Expression.GetChannel(convert(e), tpe, loc)
```

Comment:
this is unnecessary to convert the same expression twice

Reply:
They doesn't do that in arraystore

Reply:
Check Expression.Ref

Reply:
```
case Expression.GetChannel(exp, tpe, loc) =>
       val e = convert(exp)
       Expression.GetChannel(e, tpe, loc)
```
instead of
```
case Expression.GetChannel(exp, tpe, loc) =>
       val e = convert(exp)
       Expression.GetChannel(convert(e), tpe, loc)
```